### ISSUE TITLE
Re-enable podman test

### ISSUE THREAD

**Issue by thomaseizinger** [testcontainers/testcontainers-rs#299]
Unfortunately, the podman test started to hang forever. We had to disable it to unblock current work happening.

@ctron Do you know of any reason why podman would suddenly start hanging? I can reproduce the issue locally with podman version 3.3.0. Do you mind helping out here please? :)

**Comment by ctron** [comment]
Sure, I am not a podman expert, but I can try to look into it.

**Comment by ctron** [comment]
This looks like an issue with `podman logs -f` in case a container is already terminated/completed.

**Comment by thomaseizinger** [comment]
Thanks for digging into this and opening a bug report upstream!

I guess we just need to wait now until podman releases a new version and GitHub's virtual environment ships that version.

**Comment by ctron** [comment]
> I guess we just need to wait now until podman releases a new version and GitHub's virtual environment ships that version.

Yes, I hope that will be soon.

**Comment by thomaseizinger** [comment]
Closing in favor of https://github.com/testcontainers/testcontainers-rs/issues/422.
